server/db/models/User.js

```diff
@@ -43,6 +43,15 @@ module.exports = function (sequelize, DataTypes) {
         });
       },
 
+      getEmployeeEmail: function (id) {
```

Comment:
Agreed, this seemed like the simplest for now. Will make a comment about this in the User model though so we remebmer